registry/remote/repository_test.go

```diff
@@ -2203,3 +2205,53 @@ func Test_ManifestStore_FetchReference(t *testing.T) {
 		t.Errorf("Manifests.FetchReference() = %v, want %v", got, manifest)
 	}
 }
+
+// Testing fix of Issue 177 with dummy client credential
+func Test_BlobStore_Push_Fetch_Iss177(t *testing.T) {
```

Comment:
We should call out the testing scenario in the function name, such as `Test_BlobStore_Push_Port443` or something like that.